static/js/lightbox_canvas.js

```diff
@@ -41,6 +41,19 @@ const funcs = {
         meta.zoom = Math.min(Math.max(zoom, 1), meta.maxZoom);
     },
 
+    moveImage: function (meta, movementDirection) {
+        // condition to image movement by movementDirection hotkeys
+        if (movementDirection === "up") {
+            meta.coords.y = meta.coords.y + 0.05 * meta.coords.y;
+        } else if (movementDirection === "down") {
+            meta.coords.y = meta.coords.y - 0.05 * meta.coords.y;
+        } else if (movementDirection === "left") {
+            meta.coords.x = meta.coords.x + 0.05 * meta.coords.x;
+        } else if (movementDirection === "right") {
+            meta.coords.x = meta.coords.x - 0.05 * meta.coords.x;
```

Comment:
I don't think you want to be moving by 5% of the current coordinate, but instead by some fixed amount.  Otherwise, when we're at the far left, the keys won't do anything.

Reply:
The keys will not stop as the coordinates are for the center of the image. BTW should I change it to any fixed value?

Reply:
I had changed its value to a static value as I find that moving it with a certain percentage of current coordinates can make the experience worse due to more or less movement as we move away from the center. For now, I had made its value 0.05 as I want to experiment with values in multiples of 5 (they are simpler in most cases). Also after testing it visually, I thought that it suits better than other multiples.